electron/ironfish/IronFishManager.ts

```diff
@@ -516,4 +520,16 @@ export class IronFishManager implements IIronfishManager {
 
     return Promise.resolve(result)
   }
+
+  getNodeConfig(): Promise<Partial<ConfigOptions>> {
+    return Promise.resolve(this.nodeSettings.getConfig())
+  }
+
+  async saveNodeConfig(values: Partial<ConfigOptions>): Promise<void> {
+    this.nodeSettings.setValues(values)
+    await this.nodeSettings.save()
+    await this.stop()
+    await this.initialize()
```

Comment:
no need initialize app twice.

Reply:
After stopping node we need to do steps from initialize function.